source/Octopus.Tentacle.Contracts/KubernetesScriptServiceV1Alpha/StartKubernetesScriptCommandV1Alpha.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Octopus.Tentacle.Contracts.KubernetesScriptServiceV1Alpha
+{
+    public class StartKubernetesScriptCommandV1Alpha
+    {
+        public StartKubernetesScriptCommandV1Alpha(
+            ScriptTicket scriptTicket,
+            string taskId,
+            string scriptBody,
+            string[] arguments,
+            ScriptIsolationLevel isolation,
+            TimeSpan scriptIsolationMutexTimeout,
+            string isolationMutexName,
+            PodImageConfiguration? podImageConfiguration,
```

Comment:
This is a new POCO to hold the pod image configuration (image, url, feed username, password) information, rather than having 4 individual properties